src/webview/mod.rs

```diff
@@ -662,6 +664,19 @@ impl WebviewExtMacOS for WebView {
   }
 }
 
+#[cfg(target_os = "android")]
+/// Additional methods on `WebView` that are specific to Android
+pub trait WebviewExtAndroid {
+  fn handle(&self) -> JniHandle;
+}
```

Comment:
I think this should be something like to match other platforms and allow a bit more flexibility.

```suggestion
pub trait WebviewExtAndroid {
  fn jni_env(&self) -> &JNIEnv; // or *mut JNIEnv
  fn activity(&self) -> &GlobalRef;
  fn webview(&self) -> &GlobalRef
}
```

Reply:
I can make that change, but we still need some method to execute code in the context of the looper used in the android setup. The issue being that method on the webview can only be called from that thread. Calls from any other thread just result in errors.

Reply:
we could also expose a ptr to the looper where they can use https://docs.rs/ndk/latest/ndk/looper/struct.ForeignLooper.html#method.from_ptr but your approach is probably safer.

Reply:
Also, there's the actual difficulty of simply getting those references. They're owned by the MainPipe and there is not a ready reference to that object anywhere.